src/js/pre-need/config/form.js

```diff
@@ -279,7 +290,9 @@ const formConfig = {
                       'militaryStatus',
                       'isDeceased',
                       'dateOfDeath',
-                    ], veteran.properties)
+                    ], _.set('militaryStatus.enum', [
+                      'A', 'I', 'D', 'S', 'R', 'E', 'O', 'V', 'X'
```

Comment:
Are these two different lists (here and line 181)? Can we make these correct in the schema instead of overwriting them here?

Reply:
You're correct @jbalboni, this can be removed.